dist/index.js

```diff
@@ -671,7 +671,7 @@ function substituteEnvVariables(config, pipeline) {
         }
     }
     // Warn about all unsubstituted variables
-    const unsubstituted = [...pipeline.matchAll(/\{([^} ]+)\}/g)];
+    const unsubstituted = [...pipeline.matchAll(/((?<!\{)\{)[^{}|"]*(\}(?!\}))/g)];
```

Comment:
It's probably worth add a comment here so in the future people understand what is going on here. e.g. something like `Ignore variables within double brackets as those will be substituted by VIB`